persone.rb

```diff
@@ -26,4 +29,9 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(date, book)
+    rental = Rental.new(date, book, self)
+    rentals << rental unless rentals.include?(rental)
+  end
```

Comment:
- [ ] Hey! Quick note 📝: In the rental class, you're already appending the newly created person to the rentals array. 🚀 No need to push it again in the person class, as it could lead to duplication. 🚫 This adjustment ensures a streamlined process and prevents unnecessary redundancy in the rentals array. Keep up the good coding practices! 👩‍💻🔄

```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```